.cascade-code/Chapman.edu/_components/omni_nav_v2/omni_nav_v2.vtl

```diff
@@ -801,4 +804,4 @@
 ## Main OmniNav HTML Block
 ##
 #setVars()
-#buildOmniNav()
+#buildOmniNav()
```

Comment:
Just need a new empty line at the end of this file according to best practices.

Reply:
@cbryant24 thx. added empty line to end